src/test/unit/config.test.js

```diff
@@ -35,6 +37,19 @@ describe('config', () => {
         jest.resetModules();
     });
 
+    it('correctly parses OUTBOUND_ERROR_STATUSCODE_EXTENSION_KEY when set', () => {
+        process.env.OUTBOUND_ERROR_STATUSCODE_EXTENSION_KEY = outErrorStatusKey;
+        const config = require('../../config');
+        expect(config.outboundErrorStatusCodeExtensionKey).toEqual(outErrorStatusKey);
+    });
+
+    it('correctly parses OUTBOUND_ERROR_STATUSCODE_EXTENSION_KEY when NOT set', () => {
+        delete process.env.OUTBOUND_ERROR_STATUSCODE_EXTENSION_KEY;
```

Comment:
`delete ...` not needed as `process.env` get cleared in `beforeEach` and `afterEach`

Reply:
I dont agree. the code in beforeEach/afterEach does not clear the environment. If a value was already in the environment for OUTBOUND_ERROR_STATUSCODE_EXTENSION_KEY then it would not be removed.

Reply:
you're right. I see that the env is been saving in `beforeEach` and restored in `afterEach`. So I suspect that test cases run in parallel and share the same `process.env`.